lib/google/gax/api_callable.rb

```diff
@@ -277,12 +277,13 @@ def create_api_call(func, settings, params_extractor: nil)
     # @return [Proc] A proc takes the API call's request and returns
     #   an Event object.
     def bundleable(desc)
-      proc do |api_call, request, settings|
-        return api_call(request) unless settings.bundler
+      proc do |api_call, request, settings, block|
+        return api_call(request, block) unless settings.bundler
         the_id = Google::Gax.compute_bundle_id(
           request,
           desc.request_discriminator_fields
         )
+        # TODO: how to handle the block arg in next line?
```

Comment:
Bundling calls should probably raise a runtime exception if you try to get metadata through a block. There's not a 1-1 correspondence between bundled calls and gRPC calls, so getting this right is probably a little tricky.